region/brinstar/red.json

```diff
@@ -310,25 +412,41 @@
           "to": [
             {
               "id": 1,
-              "requires": null
+              "strats": [
+                {
+                  "name": "Base",
+                  "notable": false,
+                  "requires": null
+                }
+              ]
             },
             {
               "id": 5,
-              "requires": [
-                {"or": [
-                  "canUseFrozenEnemies",
-                  "canTrickyWalljump",
-                  {"and": [
+              "strats": [
+                {
+                  "name": "Use Frozen Enemies",
+                  "notable": false,
+                  "requires": ["canUseFrozenEnemies"]
+                },
+                {
+                  "name": "Red Tower Walljump Climb",
+                  "notable": true,
+                  "requires": ["canTrickyWalljump"]
+                },
+                {
+                  "name": "Space Jump",
+                  "notable": false,
+                  "requires": [
                     "SpaceJump",
                     {"enemyKill":{
                       "enemies": [
                         [ "Ripper","Ripper","Ripper","Ripper" ]
```

Comment:
Why the kill requirement?  so there is an easy way up?  And it should only require killing the top ripper.  
Additional strats: 9to5 IBJ, and node 1to5 shinespark where you use a missile to break the shot block.

Reply:
Killing just the top Ripper does seem more in line with the note saying only the last jump is considered tricky.

That IBJ 100% requires being able to bomb the blocks without losing your IBJ. I feel like that's considerable harder than an IBJ and should be a different tech?

As for that spark, I'll just say wow.